### ISSUE TITLE
z probe servo not work anet a8

### ISSUE THREAD

**Issue by ISU000** [MagoKimbra/MarlinKimbra#180]
hi, i test in all input 27,28,29,30 but with m280 don't move servo (3D Touch) the 3D Touch with marlin 3.1.6 in my anet work well but without nextion 7 lcd.

This is my configuration:
[Configuration_Overall.h.zip](https://github.com/MKFirmware/MarlinKimbra/files/1585932/Configuration_Overall.h.zip)

**Comment by MagoKimbra** [comment]
#define SERVO0_PIN 30 in configuration_overall????? WHY??? The pin is defined in configuration_pins!!

**Comment by ISU000** [comment]
Yes i know but in original input 27 in board 100 anet when i put servo cable de light 1 go on, for test i change inputs in board configuration 27,28,29,30 and 30 servo dont move, i test with normal servo and not move. In marlin 3.1.6 with anet board work perfect.

I put in overall for make test but servo not work.
This moning i test with ramps 1.4 board and work perfect. 

Thanks
Iker



El 27 dic. 2017 15:51, "MagoKimbra" <notifications@github.com> escribió:
#define SERVO0_PIN 30 in configuration_overall????? WHY??? The pin is defined in configuration_pins!!

**Comment by MagoKimbra** [comment]
Ok you must change on your board file

**Comment by ISU000** [comment]
Yes



El 27 dic. 2017 17:07, "MagoKimbra" <notifications@github.com <mailto:notifications@github.com>> escribió:
Ok you must change on your board file
